tasks/unit-tests/libs/pipeline/failure_summary_tests.py

```diff
@@ -287,14 +287,14 @@ def test_send_summary_messages(self, mock_slack: MagicMock = None):
                     )
                     for _ in range(10)
                 ],
-                # agent-ci-experience, agent-build-and-releases
+                # agent-devx-infra, agent-build-and-releases
                 *[
                     ProjectPipelineJob(
                         manager=MagicMock(), attrs={'name': 'tests_release', 'status': 'failed', 'allow_failure': False}
                     )
                     for _ in range(5)
                 ],
-                # agent-ci-experience, agent-build-and-releases
+                # agent-devx-infra, agent-build-and-releases
```

Comment:
```suggestion
                # agent-devx-infra, agent-delivery
```

Reply:
done in another PR https://github.com/DataDog/datadog-agent/pull/26965